AEFeinstein/Swadge-IDF-5.0
# Origin/lumberjack

### Description
10 levels for Panic mode
10 levels for Attack mode
Multiplayer stages for Panic and Attack mode
Full gameplay loop
Error checking for connection lost to alert user to press B to back out
Properly load Cho and Swadgeland Guy for save files

### Test Instructions
Play Lumber Jacks: Panic Single Player
Play Lumber Jacks: Attack Single Player
Play Lumber Jacks: Panic Multiplayer with two swadges
Play Lumber Jacks: Attack Multiplayer with two swadges
Play Swadgeland and see if "Guy" appears as a selectable character in Lumber Jacks
Play Magtroid Pocket and see if "Cho" appears as a selectable character in Lumber Jacks

### Ticket Links


### Readiness Checklist
